assets/scripts/streets/__tests__/StreetMetaWidthContainer.test.js

```diff
@@ -19,30 +18,33 @@ describe('StreetMetaWidthContainer', () => {
     expect(wrapper.asFragment()).toMatchSnapshot()
   })
 
-  it('updates street label on selection change', () => {
+  it.skip('updates street label on selection change', () => {
```

Comment:
Just wanna double check - are we skipping this test and the one below it on purpose?

Reply:
These are skipped on purpose for now, since we need to rewrite these tests for them to pass. (As discussed, this component's interaction is a great candidate for Cypress.)